tableauserverclient/server/request_factory.py

```diff
@@ -86,6 +86,27 @@ def update_req(self, column_item):
         return ET.tostring(xml_request)
 
 
+class DataAlertRequest(object):
+    def add_user_to_alert(self, alert_item, user_id):
+        xml_request = ET.Element('tsRequest')
+        user_element = ET.SubElement(xml_request, 'user')
+        user_element.attrib['id'] = user_id
+
+        return ET.tostring(xml_request)
+
+    def update_req(self, alert_item):
+        xml_request = ET.Element('tsRequest')
+        dataAlert_element = ET.SubElement(xml_request, 'dataAlert')
+        dataAlert_element.attrib['subject'] = alert_item.subject
+        dataAlert_element.attrib['frequency'] = alert_item.frequency
```

Comment:
this is where we have to convert to lowercase

Reply:
`.lower()` call added